helper_test.go

```diff
@@ -577,6 +577,44 @@ func createWorkspace(t *testing.T, client *Client, org *Organization) (*Workspac
 	}
 }
 
+func createWorkspaceWithVCS(t *testing.T, client *Client, org *Organization) (*Workspace, func()) {
+	var orgCleanup func()
+
+	if org == nil {
+		org, orgCleanup = createOrganization(t, client)
+	}
+
+	ctx := context.Background()
+
+	token, ocTestCleanup := createOAuthToken(t, client, org)
+	w, err := client.Workspaces.Create(ctx, org.Name, WorkspaceCreateOptions{
+		Name: String(randomString(t)),
+		VCSRepo: &VCSRepoOptions{
+			OAuthTokenID: String(token.ID),
```

Comment:
We need to specify an `Identifier` as well, but that actually needs to exist in order for the call to succeed. If you could just add a hardcoded string for now, then I will update the test once this is merged so it will take a env var instead 👍